### ISSUE TITLE
confuse about step2

### ISSUE THREAD

**Issue by JoyLuo** [KaleidoZhouYN/Details-on-Face-Recognition#14]
why start the step2 until the accuracy is 0.9? could the accuracy bigger than 0.9? for example , the accuracy of the pretrained model of the step1 is 1.

and the is no quadruple type in you finetune step

**Comment by KaleidoZhouYN** [comment]
accuracy reach 0.9 is to make sure that softmax is converging pretty well,accuracy is 1 is also ok.

double is ok for metric learning ,no need for quadruple.
